delete-user-data/functions/src/index.ts

```diff
@@ -122,10 +122,9 @@ const clearFirestoreData = async (firestorePaths: string, uid: string) => {
 
       if (!isRecursive) {
         logs.firestorePathDeleting(path, false);
-        await admin
-          .firestore()
-          .doc(path)
-          .delete();
+        await admin.firestore().runTransaction((async transaction => {
```

Comment:
Here and everywhere: Do you mind adding a comment on why you are doing this ("Wrapping in transaction to allow for automatic retries").

Reply:
Sure thing, will get them added.